cssdb.json

```diff
@@ -229,15 +212,16 @@
     "specification": "https://www.w3.org/TR/css-color-4/#color-adjust",
     "stage": 2,
     "browser_support": {
-      "and_ff": "48",
-      "firefox": "48",
-      "kaios": "2.5"
+      "and_ff": "97",
+      "firefox": "97",
```

Comment:
Very big jump. Was caniuse wrong here or were we tracking it wrong?

Reply:
https://caniuse.com/mdn-css_properties_print-color-adjust

> Uses the non-standard name: `color-adjust`